src/Drupal/Migrate/MigrateExecutable.php

```diff
@@ -259,7 +259,7 @@ protected function handleMissingSourceRows(MigrationInterface $migration): void
 
         if ($destinationIds) {
             $missingSourceEvent = new MigrateMissingSourceRowsEvent($migration, $destinationIds);
-            $this->getEventDispatcher()->dispatch($missingSourceEvent);
+            $this->getEventDispatcher()->dispatch($missingSourceEvent, MigrateMissingSourceRowsEvent::class);
```

Comment:
@chrisolof, thank you for this catch. I think it should work also without the 2nd parameter. Could you try?

Reply:
Hm, I think I’m wrong. Will look during the weekend